depends/packages/rustcxx.mk

```diff
@@ -0,0 +1,13 @@
+package=rustcxx
+$(package)_version=$(native_cxxbridge_version)
+$(package)_file_name=$(native_cxxbridge_file_name)
+$(package)_sha256_hash=$(native_cxxbridge_sha256_hash)
```

Comment:
Should this have `$(package)_dependencies=native_cxxbridge`?

Reply:
What that does is two things:
- Ensure the dependency package is processed before the current package (IIRC).
- Extract the dependency package before processing the current package.

We don't need the latter, because the file we want isn't the `cxxbridge` binary (though it _could_ be in theory, as we could alternatively generate this via `cxxbridge --header -o path/to/rust/cxx.h`).

We also don't need the former, because native packages are always loaded before non-native packages. We don't have an expressed dependency like this for `libcxx` either, for comparison.